_includes/selling-points.html

```diff
@@ -0,0 +1,48 @@
+<section class="layout">
+  <section class="content content--large content--grid content--centered">
+    <article class="content--grid__item">
+      <img src="images/icon-ruby.svg" alt="Keep your rails workflow" />
+      <h4>
+        Keep your Rails workflow
+      </h4>
+      <p>
+        Leverage the asset pipeline, and take advantage of all the
+        conveniences of Rails.
```

Comment:
@jasonramirez what do you think of

> Serve both your Ember and Rails applications from a single process.

After the changes made in https://github.com/thoughtbot/ember-cli-rails/pull/250 and some plans I have, we're going to be moving away from the asset pipeline.